docs/installation.rst

```diff
@@ -70,3 +70,36 @@ Once your configuration is done, you can launch the deployment :
 .. code-block:: bash
 
     (venv) $ enos deploy
+
+
+The deployment is the combination of the following three phases:
+
+1. Acquire the raw resources that are necessary for the deployment of
+   OpenStack. Enos acquires resources according to the ``provider``
+   and ``resources`` information in the reservation file. Someone can
+   perform this phase by calling ``enos up``.
+
+2. Deploy OpenStack to the resources acquired during the previous
+   phase. Enos uses the resource list provided by the previous phase
+   and combines it with the information specified in the file targeted
+   by the ``inventory`` key to produce a file that gives a mapping of
+   which OpenStack services have to be deployed to which resources.
+   Enos then calls the Kolla-Ansible tool with this file to deploy the
+   containerized OpenStack services to the right resources. Someone
+   can perform this phase by calling ``enos os``.
+
+   .. note::
+
+      If you don't provide an ``inventory`` in your current working
+      directory, then Enos uses a default one. You can view it on
+      GitHub at `enos/inventories/inventory.sample
+      <https://github.com/BeyondTheClouds/enos/blob/232cd4710a84bc12f67dbc7306c9463d1e61014e/enos/inventories/inventory.sample>`_.
```

Comment:
Use the `conf.version` to generate the link

Reply:
I don't get it.

Reply:
:)
The idea is to make sure the file pointed here stick to the right version.
A way to do it is to use the `conf.version` variable which will correspond to tag in git.

Reply:
Yes, but how am I supposed to used that.

Reply:
One way to do it is to use a `rst_epilog` : http://www.sphinx-doc.org/en/stable/config.html#confval-rst_epilog

something like : 
`rst_epilog = ".. |my_url| replace:: https://github.com/%s/foo/bar" % (version)`
will allow you to use `|my_url|` instead of the raw link.

Reply:
I put the following in the conf.py: `rst_epilog = '.. |enos_version| replace:: %s' % VERSION` and then put |enos_version| in the url but the variable is not interpolated.

Reply:
I don't know what prevent the variable to be interpolated. But this should be fine if you create the full link in the `rst_epilog`

Reply:
I manage to fix that with the `sphinx.ext.extlinks` extension.